src/newrelic/utilization/utilization_hash_test.go

```diff
@@ -276,10 +296,6 @@ func runUtilizationCrossAgentTestcase(t *testing.T, tc utilizationCrossAgentTest
 
 	err := json.Unmarshal(tc.ExpectedOutput, &v)
 
-	if err == nil && v.MetadataVersion != 3 {
```

Comment:
Thanks for getting this up!

Why was this check removed?

Reply:
It was a filter to prevent any v5 cross agent tests from being run when the agent only supported v3.  Now that v5 is enabled we can run all the tests.

Reply:
Is the err checked for nil later?

Reply:
The case where err != nil was not being handled previously so I broke that out first to help debugging when the test case definition JSON has an error in it.  
I also changed the metadata version check to skip any test cases requiring a version greater than is supported by the daemon to future-proof it some.